Tools/CISupport/ews-build/steps_unittest.py

```diff
@@ -7958,7 +7958,7 @@ def test_success(self):
         self.setProperty('github.number', '1234')
         self.setProperty('github.base.ref', 'main')
         self.setProperty('github.head.ref', 'eng/pull-request-branch')
-        self.setProperty('valid_reviewers', ['WebKit Reviewer', 'Other Reviewer'])
+        self.setProperty('valid_reviewers', ['WebKit Reviewer', 'Other Reviewer', 'Another Reviewer'])
```

Comment:
(the thing I said offline about losing the 2 reviewer case)

Reply:
Made sure we have a bunch of test coverage in https://github.com/WebKit/WebKit/commit/072952d31db9eb3c02a726b1e4bdcdde6aaebf1b